python/cudf/cudf/core/series.py

```diff
@@ -1288,11 +1277,9 @@ def __repr__(self):
                 length=show_dimensions,
                 na_rep=cudf._NA_REP,
             )
-        else:
             output = preprocess.to_pandas().__repr__()
 
         lines = output.split("\n")
```

Comment:
This change is causing pytest failures - Can you verify these changes locally by running pytests?

Reply:
Ooof, I think I deleted that by mistake. Thanks for the catch!